Programs/_testembed.c

```diff
@@ -196,6 +195,88 @@ static int test_bpo20891(void)
 }
 
 
+static int test_atexit_callbacks(void) {
```

Comment:
Is it actually necessary to use C code for this? As opposed to calling `support.run_in_subinterp` from `test_atexit` (as already done in two other tests).

Reply:
Awesome, didn't know about this thing, it is rewritten in Python now.